engine-types/src/lib.rs

```diff
@@ -24,8 +24,8 @@ mod v0 {
         vec::Vec,
     };
     pub use core::{
-        cmp::Ordering, fmt::Display, marker::PhantomData, mem, ops::Add, ops::Div, ops::Mul,
-        ops::Sub, ops::SubAssign,
+        cmp::Ordering, fmt::Display, marker::PhantomData, mem, ops::Add, ops::AddAssign, ops::Div,
+        ops::Mul, ops::Sub, ops::SubAssign,
```

Comment:
Is this used?

Reply:
The diff looks a little strange because of the `cargo fmt` wrapping. But the important change is the addition of `ops::AddAssign`. That trait is now implemented by `EthGas` so that I can use `+=` on the cost of the xcc precompile.